Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -126,11 +126,15 @@ - (void)createViews {
     _phoneNumberLabel = [UILabel new];
     _phoneNumberLabel.textColor = [UIColor ows_darkGrayColor];
     _phoneNumberLabel.font = [UIFont ows_regularFontWithSize:20.f];
+    _phoneNumberLabel.numberOfLines = 2;
+    _phoneNumberLabel.adjustsFontSizeToFitWidth = YES;
```

Comment:
I intentionally left-aligned it since it was more than one line and presented as a sentence (with a ".").

Do you think it would look better centered (and without a period)?